.devcontainer/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
```

Comment:
@SteveL-MSFT, your last commit had 3 failures in `PowerShell-CI-macos`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @(@{InputObject=System.Diagnostics.DiagnosticSource.dll; SideIndicator=<=}, @{InputObject=System.Net.NameResolution.dll; SideIndicator=<=}, @{InputObject=System.Private.Xml.Linq.dll; SideIndicator=<=}, @{InputObject=System.Xml.XDocument.dll; SideIndicator=<=}).
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 124
124:         $diffs | Should -BeExactly $null
```
Invoke-WebRequest tests.Denial of service.Image Parsing
```powershell
Expected the actual value to be greater than 5, but got 0.330475306456397.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1932
1932:             $pathologicalRatio | Should -BeGreaterThan 5
```
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '127.0.0.1', but got 10.79.10.245.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 257
257:             $result[0].TargetAddress | Should -BeExactly $realAddress
```